### DISCUSSION TITLE
The material is white around the texture

### DISCUSSION THREAD

**Discussion by Huiqqqqqq** [community/community#141444] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I put the material on the object and in addition to the texture, everything else is white
![Безымянный](https://github.com/user-attachments/assets/76bea39f-ee3e-49ce-8998-048defee1db3)

**Comment by Akash1134** [comment]
Hi @Huiqqqqqq ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as this one. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
